packages/lesswrong/lib/allComponentsVite.ts

```diff
@@ -0,0 +1,1277 @@
+import { importComponent } from './vulcan-lib';
+
+/**
+ * This file registers each component-containing file with a call to
```

Comment:
This comment seems like it's coming from the original components file, and needs to be updated